packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -1519,8 +1519,9 @@ class FullRemixableTracks(Resource):
     @cache(ttl_sec=5)
     def get(self):
         args = track_remixables_route_parser.parse_args()
+        current_user_id = get_current_user_id(args)
         args = {
-            "current_user_id": get_current_user_id(args),
+            "current_user_id": current_user_id,
```

Comment:
these are all from an earlier commit that I didn't think was necessary to fully revert. should be no-op